terraform_files/baremetal/main.tf

```diff
@@ -152,8 +152,8 @@ data "libvirt_network_dns_host_template" "api" {
 }
 
 data "libvirt_network_dns_host_template" "api-int" {
-  count    = var.bootstrap_in_place ? 1 : 0
-  ip       = var.single_node_ip
+  count    = 1
+  ip       = var.bootstrap_in_place ? var.single_node_ip : var.api_vip
```

Comment:
only relevant to api-int? or should be set for all those ``*.apps.*`` entries?

Reply:
yeah, I'm not explicitly asking you to do touch it (because I also don't know)
I'm just asking what are the changes it might make sense for those other parts, so that if we'll have problems in the CI I'll know how to mitigate that

Reply:
So "know idea" is also a good answer :)